aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -271,6 +275,18 @@ def parse_xform_to_avro_schema(xml_definition, default_version=DEFAULT_XFORM_VER
         current_type = definition.get('type')
         current_name = xpath.split('/')[-1]
         current_doc = definition.get('label')
+        select_options = None
```

Comment:
why don't you include all of this inside the `__get_xform_instance_skeleton` method and return another property like `choices`?